Adapt Office 365 dashboard documentation (#7275)
wazuh/wazuh-documentation: source/cloud-security/office365/monitoring-office365-activity.rst

Replace ``<YOUR_TENANT_ID>``, ``<YOUR_CLIENT_ID>``, and ``<YOUR_CLIENT_SECRET>`` with the organization's credentials for the tenant.

Visualizing the Office 365 activity

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: **nitpick**: the underline size should be the same count of characters of the title. I think this emits a warning in the building time.

Visualizing the Office 365 activity
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The Wazuh dashboard has an Office 365 module that provides detailed information and insights about the events that occur in Office 365. The module is disabled by default. You have to enable the module explicitly to make use of the tailored visualization.

Perform the following steps to enable the Wazuh module for Office 365 on the Wazuh dashboard.

#. Navigate to the Wazuh dashboard and click on **Wazuh** > **Settings** > **Modules**.

   .. thumbnail:: /images/cloud-security/office365/wazuh-settings-module.png
      :title: Wazuh settings module
      :alt: Wazuh settings module
      :align: center
      :width: 80%

#. Enable the Wazuh module for Office 365 in the **Security Information Management** section.

>    .. thumbnail:: /images/cloud-security/office365/enable-office365-module.png

Reviewer 2: We could remove the `/images/cloud-security/office365/enable-office365-module.png` file from the repository since it's no longer needed.

> The Wazuh module for Office 365 provides three visualization options to gain better insight into the events that occur in Office 365. These visualization options are the **Panel**, **Dashboard**, and **Events**. Navigate to the  **Office 365** tab on the **Cloud security** section of the Wazuh dashboard to view this information.

Reviewer: **suggestion**: the order of tabs was changed, maybe we could change it here too.

> Panel

Reviewer 2: We could move the `Panel` sub-section down below the `Dashboard` sub-section to match the enumeration order above and the tabs order visible in the image.

Replace ``<YOUR_TENANT_ID>``, ``<YOUR_CLIENT_ID>``, and ``<YOUR_CLIENT_SECRET>`` with the organization's credentials for the tenant.

Visualizing the Office 365 activity
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The Wazuh module for Office 365 provides three visualization options to gain better insight into the events that occur in Office 365. These visualization options are the **Dashboard**, **Panel**, and **Events**. Navigate to the  **Office 365** tab on the **Cloud security** section of the Wazuh dashboard to view this information.

Reviewer 2: ```suggestion
The Wazuh dashboard has an Office 365 module that provides detailed information and insights about the events that occur in Office 365. The module provides three visualization options.

-  **Dashboard**
-  **Panel**
-  **Events**

To select any of them, navigate to the  **Office 365** tab on the **Cloud security** section of the Wazuh dashboard.
```

   :align: center
   :width: 80%

> 

Reviewer 2: ```suggestion
```
